### DISCUSSION TITLE
How to run flow simulation without  optimizations in DAFoam?

### DISCUSSION THREAD

**Discussion by jh36714753** [mdolab/dafoam#332] [General]
Hi, I am very interested in running Rotor 37 flow simulation in DAFoam, but I do not want any design optimization.

How can I run Rotor 37 compressor flow simulation without optimizations in DAFoam? Where can I find some references about it?

Specifically, I want to use the solver "DATurboFoam" to simply simulate the flow fields.I tried to run Rotor 37 flow simulation in OpenFOAM but I failed. So I want to run it in DAFoam.

Thanks!

**Comment by jh36714753** [comment]
It is very hard to simulate Rotor 37 in OpenFOAM (even in foam-extend), so I am trying to run it in DAFoam.

**Comment by friedenhe** [comment]
Check: https://github.com/DAFoam/validations/blob/main/RAE2822_Airfoil/RAE2822_Medium_SA/runScript.py

**Reply by jh36714753** [reply]
Thank you very much, I will check it carefully.

**Comment by shaheidula** [comment]
Hi, have you tried this case?
